dotCMS/src/enterprise/java/com/dotcms/enterprise/publishing/remote/handler/ContentHandler.java

```diff
@@ -769,8 +811,11 @@ private void saveContent(Contentlet content, final File folderOut, final User us
 
 		//Verify if this contentlet is a FileAsset
         if (BaseContentType.FILEASSET.equals(content.getContentType().baseType())) {
-            // Wiping out the thumbnails and resized versions
-            APILocator.getFileAssetAPI().cleanThumbnailsFromContentlet(content);
+			Folder testFolder = APILocator.getFolderAPI().find(content.getFolder(), APILocator.systemUser(), false);
+			if(testFolder != null && testFolder.getInode() != null) {
```

Comment:
why are you validating if the folder is null here, if inside the `cleanThumbnailsFromContentlet` the same validation is done?

Reply:
done